3_Risk.md

```diff
@@ -1,12 +1,28 @@
 # Metric: Risk
 
+## Human Factors
 Activity Metric | Description
 --- | ---
 Contributor Importance | Percentage of commits by individual contributors from identified organizations over time
 Qualified Committers | Contributions over time and what components they commit to over time
 User Dependency | Number of users who are aware that they depend on the software over time
 Paid developers | Number of paid developers in community over time
 
+## Copyright and License Factors
+ Activity Metric | Description
+ --- | ---
+Copyright Declaration | Copyright declaration on software package
+Copyright Text | Copyright text provided 
+Package License Declaration | License declaration on software package
+File License Declarations | License declarations on software package files
+License Identification Methods | Declaration of methods used to identify software package licenses

```

Comment:
What is "License Identification Methods" exactly? Whether a method was declared with which licenses are identified? Or is this "Method with which licenses are declared. Possible values: full license text; SPDX-Identifier."?